Public/Start-Polaris.ps1

```diff
@@ -50,7 +52,7 @@ function Start-Polaris {
         Use-PolarisJsonBodyParserMiddleware -Polaris $Polaris
     }
 
-    $Polaris.Start( $Port )
+    $Polaris.Start( $Port, [bool]$Https )
```

Comment:
I don't think you need to create the cert. You should be able to just ping that the url is accessible. It might complain that there's no valid cert, but at least the url will resolve. Does that make sense?

Reply:
I don't think that it will work at all without some kind of SSL binding for the port. The request dies at the TLS handshake. I believe it is because it is netsh handling the handshake and not HTTPlistener.

 Currently I am working on trying to start polaris with HTTPS without admin. I am getting an access is denied error. I believe it has something to do with netsh urlacl's. The docs for that are [here](https://docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/configuring-http-and-https). I also have been trying it on Linux with stunnel, no luck so far.

Reply:
Fair enough. No need for a test in that case. We can have HTTPS only be activated when running as admin. You can see in the code I had to do similar things when trying to use `http://+` vs `http://localhost`